docs/Examples/ReactiveSwift.md

```diff
@@ -0,0 +1,23 @@
+# ReactiveSwift examples
+
+ReactiveSwift extension to Moya is added via `reactive` property of `MoyaProvider. As in normal setup,
```

Comment:
Was doing my own CR and didn't want to make multiple commits with code review fixes, so waited till someone did it as well so I can cumulate 😄